<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="QIN5CD04618ZL" failures="2" tests="2" name="api.PlayerCreateWithUserEditorNegativeTest" time="0.752" errors="0" timestamp="2025-08-10T20:08:30 TRT" skipped="0">
  <testcase classname="api.PlayerCreateWithUserEditorNegativeTest" name="testUserCannotAccessAdminOperations" time="0.371">
    <failure message="Response body is empty; cannot map to ErrorBody" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Response body is empty; cannot map to ErrorBody
at api.client.ResponseWrapper.readError(ResponseWrapper.java:56)
at api.PlayerCreateWithUserEditorNegativeTest.assertCreatePlayerWithAuthorizationError(PlayerCreateWithUserEditorNegativeTest.java:96)
at api.PlayerCreateWithUserEditorNegativeTest.testUserCannotAccessAdminOperations(PlayerCreateWithUserEditorNegativeTest.java:72)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- testUserCannotAccessAdminOperations -->
  <testcase classname="api.PlayerCreateWithUserEditorNegativeTest" name="testCreatePlayerWithUserEditor" time="0.381">
    <failure message="Response body is empty; cannot map to ErrorBody" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Response body is empty; cannot map to ErrorBody
at api.client.ResponseWrapper.readError(ResponseWrapper.java:56)
at api.PlayerCreateWithUserEditorNegativeTest.assertCreatePlayerWithAuthorizationError(PlayerCreateWithUserEditorNegativeTest.java:96)
at api.PlayerCreateWithUserEditorNegativeTest.testCreatePlayerWithUserEditor(PlayerCreateWithUserEditorNegativeTest.java:62)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- testCreatePlayerWithUserEditor -->
</testsuite> <!-- api.PlayerCreateWithUserEditorNegativeTest -->
